Index: TeamCode/src/main/java/developing/TestAuto.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- TeamCode/src/main/java/developing/TestAuto.java	(date 1616895471000)
+++ TeamCode/src/main/java/developing/TestAuto.java	(date 1616895471000)
@@ -0,0 +1,19 @@
+package developing;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
+import autofunctions.Path;
+
+public class TestAuto extends LinearOpMode {
+    TestRobot bot = new TestRobot();
+    Path path = new Path(0,0,0);
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+        initialize();
+    }
+
+    public void initialize() {
+        bot.init(hardwareMap);
+    }
+}
